src/state/user/hooks/index.tsx

```diff
@@ -20,8 +20,9 @@ import {
   updateUserFarmStakedOnly,
   updateUserSingleHopOnly,
   updateUserSlippageTolerance,
+  updateGasPrice,
 } from '../actions'
-import { deserializeToken, serializeToken } from './helpers'
+import { deserializeToken, GAS_PRICE_GWEI, serializeToken } from './helpers'
```

Comment:
Unused variable

Reply:
Ah Sorry Hachi - all of this _is_ used, it's just commented atm. Needed to remove the 10 GWEI testnet gas price override so that Cheems could test the gas price is being properly set on the farm auctions txs

Reply:
All reinstated